handler.go

```diff
@@ -343,6 +362,18 @@ func (fm *Frontman) FetchInput(inputFilePath string) (*Input, error) {
 }
 
 func (fm *Frontman) Run(inputFilePath string, outputFile *os.File, interrupt chan struct{}) {
+	// Update statistics about uptime
+	// TODO: Is there any chance Run can be alled multiple times?
+	go func() {
+		for {
+			fm.Stats.Uptime++
```

Comment:
Maybe it will be better just to save the time when frontman was started?

Reply:
yeah... agree :)